Avatar sizes increase to 48px globally.

ishammahajan opened the pull request:
Changes the avatar sizes found everywhere in the application to 48px. Since this is the size used by most other comparable messaging applications, 48px seems like a good option to go for (other reasons explained in the commit/s).

In the first commit `.message-brief` is set to `padding: 0 16px 16px 73px`. While this 73 might seem like an arbitrary number, it's confusing to me and I don't know why this is the case, but that is the only one which makes the messages perfectly aligned.

ishammahajan commented:
(The build failing doesn't have anything to do with the changes made in this code, it seems like. My other PRs also have the exact same error happening.)

Screenshots:

![Screenshot_20190618-232202](https://user-images.githubusercontent.com/33200024/59707471-51c53c00-9220-11e9-87a2-5ea538c1e735.jpg)
![Screenshot_20190618-232126](https://user-images.githubusercontent.com/33200024/59707476-538eff80-9220-11e9-8250-32cdeddd917a.jpg)
![Screenshot_20190618-232159](https://user-images.githubusercontent.com/33200024/59707486-5a1d7700-9220-11e9-8d8c-91e8fe377496.jpg)
![Screenshot_20190618-232038](https://user-images.githubusercontent.com/33200024/59707487-5b4ea400-9220-11e9-995e-1f9e0c4823c0.jpg)

gnprice commented:
> (The build failing doesn't have anything to do with the changes made in this code, it seems like. My other PRs also have the exact same error happening.)

(Yup -- see also #3526 .)

gnprice commented:
> In the first commit `.message-brief` is set to `padding: 0 16px 16px 73px`. While this 73 might seem like an arbitrary number, it's confusing to me and I don't know why this is the case, but that is the only one which makes the messages perfectly aligned.

Huh, puzzling. That shouldn't be.

And in fact, when I run this, it looks slightly misaligned to me. Here's a screenshot, super zoomed in (a feature of the Android emulator -- Ctrl+Z, or the "zoom" icon in the emulator toolbar):
![image](https://user-images.githubusercontent.com/28173/59810573-baf98d80-92ba-11e9-9e74-7bb724f6992d.png)

The PM conversations screen looks great, though!

gnprice commented:
Oh wow, here's a clue about the alignment issue, though -- in an emulated Nexus 5, the avatar winds up being some funny number of pixels wide! Here's the `.avatar` element:
![image](https://user-images.githubusercontent.com/28173/59810899-46bfe980-92bc-11e9-9f1a-12992fcce477.png)

Not 48px, but rather 38.828px.

(The actual avatar, the `img`, then gets scaled to that width and the corresponding height.)

I'm not sure what's controlling that... but that will be something for you to debug! :wink: 

One related odd thing I notice: we set `min-width`. If that's doing anything at all, either before or after this change, then it's probably at least a clue as to what's going on here.

gnprice commented:
In the second commit:
```
@@ -42,7 +42,7 @@ class UserAvatarWithPresence extends PureComponent<Props> {
   static defaultProps = {
     avatarUrl: '',
     email: '',
-    size: 32,
+    size: 48,
```

Why change the default?

Are there other call sites this affects? If so, we'll want to look and make sure the effect is something we want.

ishammahajan commented:
> One related odd thing I notice: we set `min-width`. If that's doing anything at all, either before or after this change, then it's probably at least a clue as to what's going on here.

You're right! I don't know why, but setting `min-width: 48` sets the width properly. I suspect it has something to do with the width of the phone (and hence the viewport), because on my device the width of the avatar before I changed `min-width` was ~ 40.821px. Something to do with [used value](https://developer.mozilla.org/en-US/docs/Web/CSS/used_value) in css, I think.

Checking [width](https://developer.mozilla.org/en-US/docs/Web/CSS/width) on MDN -
> The min-width and max-width properties override width.

Which doesn't appear to be true, because if you remove the `width` property, the avatars become huge -- I believe the uploaded picture's width, and consequently the height (which is set to 48px!).  In any case, setting width to a value should set the width of the element in question as that value, without consideration! It's rather strange; I'm not able to figure out how the CSS works here.

> Why change the default?
> 
> Are there other call sites this affects?

Shouldn't have changed the default. It does affect multiple sites, but at all call sites, for whatever reason, we override the default (to the same value as the default 😆). I've removed the change in the default size.

gnprice commented:
Thanks @ishammahajan for the PR and these updates! Looks good -- just merged.
